phpstan.ddtrace.neon

```diff
@@ -0,0 +1,14 @@
+parameters:
+	ignoreErrors:
+		-
+			message: '#Call to an undefined method object::isHostRoot\(\)#'
+			path: src/DDTrace/Encoders/SpanEncoder.php
```

Comment:
Can you defend this one a bit? Looking briefly at the file it seems this is legitimate so I'm curious what changed such that it's reported now and needs suppressed.

Reply:
This is a very good point. Looking into it, as I had followed the wrong path before while deciding to accept the message

Reply:
This was a leftover, this error is not there anymore